coalib/bearlib/languages/__init__.py

```diff
@@ -17,17 +17,22 @@
 from .definitions.Java import Java
 from .definitions.JavaScript import JavaScript
 from .definitions.JSP import JSP
+from .definitions.m4 import m4
 from .definitions.Matlab import Matlab
 from .definitions.Markdown import Markdown
 from .definitions.ObjectiveC import ObjectiveC
 from .definitions.PHP import PHP
 from .definitions.PLSQL import PLSQL
+from .definitions.PowerShell import PowerShell
 from .definitions.Python import Python
 from .definitions.Ruby import Ruby
 from .definitions.Scala import Scala
 from .definitions.Swift import Swift
+from .definitions.Tcl import Tcl
 from .definitions.Vala import Vala
 from .definitions.TypeScript import TypeScript
+from .definitions.Unknown import Unknown
```

Comment:
Move unknown to top. It is special.

And move these editions to a separate commit, so we can ack that bit and then work on the rest of the patch.

Reply:
Unknown was already imported, didn't see at first, removing this extra import